tests/04_api/test_admin_api.py

```diff
@@ -155,6 +184,52 @@ async def test_view_workflow(
         assert res.status_code == 200
         assert len(res.json()) == 3
 
+        res = await client.get(
+            f"{PREFIX}/workflow/?timestamp_created_min="
+            f"{quote('2000-01-01T01:01:01')}"
+        )
+        assert res.status_code == 422
+        assert "timezone" in res.json()["detail"]
+
+        res = await client.get(
+            f"{PREFIX}/workflow/?timestamp_created_min="
+            f"{quote('2000-01-01T01:01:01Z')}"
+        )
+        assert res.status_code == 200
+        assert len(res.json()) == 4
+
+        res = await client.get(
+            f"{PREFIX}/workflow/?timestamp_created_min="
+            f"{quote('2000-01-01T01:01:01+00:00')}"
```

Comment:
Let's duplicate this test, and have the same API call with an explicit query parameter (that is, the one generated by `quote`).